### ISSUE TITLE
HF YituTechConvBert accuracy test is flaky for training under max-autotune mode

### ISSUE THREAD

**Issue by shunting314** [pytorch/pytorch#130172]
### 🐛 Describe the bug

The pass status for YituTechConvBert is flaky for training in max-autotune mode.

<img width="1158" alt="Screenshot 2024-07-05 at 1 21 43 PM" src="https://github.com/pytorch/pytorch/assets/52589240/40d5903b-6eaa-4693-8979-f0d288a2086e">

I've seen this again in a recent run with max-autotune [link](https://hud.pytorch.org/benchmark/compilers?dashboard=torchinductor&startTime=Fri%2C%2028%20Jun%202024%2006%3A53%3A45%20GMT&stopTime=Fri%2C%2005%20Jul%202024%2006%3A53%3A45%20GMT&granularity=hour&suite=torchbench&mode=training&dtype=amp&lBranch=gh/shunting314/162/head&lCommit=cb236e8c198b54901e4fb19698f91be786f72e25&rBranch=main&rCommit=4ee1cb9b955fcc5d75a421b19393998122136f2c)

Tried command
```
TORCHINDUCTOR_MAX_AUTOTUNE=1 time python benchmarks/dynamo/huggingface.py --accuracy --no-translation-validation --training --amp --backend inductor --device cuda --only YituTechConvBert
```
but can not repro locally.

### Versions

.

cc @ezyang @anijain2305 @chauhang @penguinwu

**Comment by masnesral** [comment]
@shunting314 Is this still an issue? Look like it's a "pass" on the pt2dash?

**Comment by shunting314** [comment]
i also see it consistently pass in the last 3 months. Close for now.
